### PR TITLE
Remove Scala 2.11 for Akka 2.6

### PR DESCRIPTION
**Pull request by chbatey**
Don't merge until we've branched.
Update jenkins jobs to not build with 2.11

### CODE REVIEW

**Review by raboof — APPROVED** [review]
We should also perhaps merge the `scala-2.13-` and `scala-2.12` source directories - but that can be done in another issue/PR

**Review by johanandren — COMMENTED** [review]
I see a few more 2.11 things when searching for "2.11" and 11 in the project sources.

AkkaDisciplinePlugin.scala:79 BehaviorSpec.scala:176, RELEASING.md

Maybe this is also the PR where we should introduce the migration docs saying that 2.11 is no longer supported and that users have to upgrade?

**Comment by chbatey** [comment]
:+1: will add the migration docs

**Comment by chbatey** [comment]
I'll issuify the comment in the RELEASING.md as it we may do something

File: RELEASING.md [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
 
-Releasing Akka requires running on at least JDK 9, but also having JDK 8
+Releasing Akka requires running on JDK 11, but also having JDK 8
 installed. The reason for this is that we want the Akka artifacts to be
-usable with JRE 8, but also want to compile some classes with JDK9-specific
+usable with JRE 8, but also want to compile some classes with JDK11-specific
 types.
 
-When we stop supporting Scala 2.11 we might be able to update the build towork
+In the future we might be able to update the build to work
```
Comment (helena) [inline RELEASING.md]: :-)

**Review by helena — APPROVED** [review]
LGTM

**Pull request merged**